pages/user.js

```diff
@@ -36,12 +33,44 @@ export async function getServerSideProps({ req }) {
 }
 
 export default function TutorialsPage({ userTutorials, userName }) {
+  const [showModal, setShowModal] = useState(false);
+  const [tutorialToBeDeleted, setTutorialToBeDeleted] = useState("");
+  const router = useRouter();
+
+  function deleteDialoge(id) {
+    setShowModal(true);
+    setTutorialToBeDeleted(id);
+  }
+
+  async function deleteTutorial() {
+    const response = await fetch(`/api/tutorials/${tutorialToBeDeleted}`, {
+      method: "DELETE",
+    });
+    await response.json();
+    router.push("/user");
+  }
+
   return (
     <>
+      {showModal && (
+        <Modal click={() => setShowModal(false)}>
+          <TbTrash style={{ fontSize: "1.5em" }} />
+          <Paragraph>Do you really want to delete this tutorial?</Paragraph>
+          <ButtonContainer>
+            <Button isPrimary>
+              <TbX style={leftIconStyle} /> no, cancel
+            </Button>
+            <Button onClick={deleteTutorial}>
+              yes, delete <TbCheck style={reftIconStyle} />
```

Comment:
Am Satzanfang bitte Groß- und Kleinschreibung beachten.

Reply:
Thank you for pointing this out. I have taken your suggestion as an opportunity to make this consistent throughout the app.